libmscore/glissando.cpp

```diff
@@ -220,7 +220,11 @@ void Glissando::layout()
 
       // FINAL SYSTEM-INITIAL NOTE
       // if the last gliss. segment attaches to a system-initial note, some extra width has to be added
-      if (cr2->segment()->measure() == cr2->segment()->system()->firstMeasure() && cr2->rtick() == 0)
```

Comment:
This seems weird.. If a CR is a gracenote, calling segment() on it will get its parent (a chord I believe) and cast it to segment, (see ChordRest::segment()) calling ->measure() will probably give weird result.